src/MeetingsSDKAdapter.js

```diff
@@ -169,6 +169,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
   getLocalMedia(ID) {
     logger.debug('MEETING', ID, 'getLocalMedia()', ['called with', {ID}]);
     const {sendAudio, sendVideo} = mediaSettings;
+    const hasVideoPermission = !navigator.userAgent.includes('iPhone OS 15_1');
```

Comment:
the word "permission" is already used for the user provided permission (allows camera access, denies or ignores the prompt). I would suggest using a different wording, eg. canUseCamera or videoSupported.